registry/unit.go

```diff
@@ -61,8 +62,48 @@ func (r *EtcdRegistry) getUnitByHash(hash unit.Hash) *unit.UnitFile {
 		}
 		return nil
 	}
+	return r.unitFromEtcdNode(hash, resp.Node)
+}
+
+// getAllUnitsHashMap retrieves from the Registry all Units and returns a map of hash to Unit
+func (r *EtcdRegistry) getAllUnitsHashMap() map[string]*unit.UnitFile {
```

Comment:
Just `getAllUnits`? Also using `get` as a prefix is not really go-like, but I see the fleet codebase is using this throughout.